tests/test_jinja.py

```diff
@@ -48,6 +48,26 @@ def test_raw_function_vault():
     assert content == "this came from the vault"
 
 
+def test_jinja_string_timestamp_fromtimestamp():
+    result = fromtimestamp("1633981527", "UTC")
+    assert result == "2021-10-11T19:45:27+00:00"
+
+
+def test_jinja_string_timestamp_fromtimestamp():
```

Comment:
these two tests have the same name, i think this means that the first test won't run?